Unity-Developer-Community/UDC-Bot
# Publish feed messages in announcement channels

### To test
- [ ] Have bot send message from RSS feed inside a normal channel. => Normal behavior
- [ ] Have bot send message from RSS feed inside an announcement channel. => Message should be published.

You can check if message has been published like this: 
![image](https://user-images.githubusercontent.com/1756398/102501477-c54a4280-407d-11eb-86c4-277cc9220521.png)

Remake of https://github.com/Sirush/UDHBot/pull/110
